internal/handler/handlers.go

```diff
@@ -2498,6 +2499,26 @@ func ApplyForJobPageHandler(svr server.Server, jobRepo *job.Repository) http.Han
 	}
 }
 
+func SaveJobPageHandler(svr server.Server, savedJobsRepository *savedJobs.Repository) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		jobID, err := strconv.Atoi(r.FormValue("jobID"))
+		if err != nil {
+			svr.Log(err, "unable to parse string to int")
+			svr.JSON(w, http.StatusBadRequest, err.Error())
+			return
+		}
+		developerID := r.FormValue("developerID")
```

Comment:
can you do an xhr request here instead of form-url-encoded? we do xhr/ajax request almost everywhere at the minute